pkg/da/data_availability_header.go

```diff
@@ -6,7 +6,7 @@ import (
 	"fmt"
 
 	"github.com/celestiaorg/celestia-app/pkg/appconsts"
-	"github.com/celestiaorg/celestia-app/pkg/wrapper"
+	nmtwrapper "github.com/celestiaorg/celestia-app/pkg/wrapper"
```

Comment:
[not blocking comment]
I like this change, but we use wrapper everywhere else. Not going to block on this, but perhaps in the future we should rename the package to force this across all repos

Reply:
I'll create an issue for this: https://github.com/celestiaorg/celestia-app/issues/1069

Reply:
since #1069 is unplanned, do you mind reverting this?

Reply:
done